extensions/Worker.Extensions.EventHubs/release_notes.md

```diff
@@ -4,6 +4,6 @@
 - My change description (#PR/#issue)
 -->
 
-### Microsoft.Azure.Functions.Worker.Extensions.EventHubs <version>
+### Microsoft.Azure.Functions.Worker.Extensions.EventHubs 5.4.0-preview1
 
-- Add `BindingCapabilities` attribute to EventHubTrigger to express function-level retry capabilities. (#1457)
\ No newline at end of file
+- Add support for binding to `EventData`(#<PR>)
```

Comment:
Is that meant to be this PR# or the one where we added the constructor?

Reply:
I need to update to include this PR.